db/R/dbfiles.R

```diff
@@ -253,7 +254,7 @@ dbfile.posterior.insert <- function(filename, pft, mimetype, formatname, con, ho
 ##'
 ##' This will check the dbfiles, inputs, machines and formats tables to see if the
 ##' file exists
-##' @name dbfile.check
+##' @name dbfile.posterior.check
```

Comment:
Sorry, you're OK. Didn't realize the amount of code Github was hiding from me between this line and the next chunk